shuffle/cli/src/account.rs

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) The Diem Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+use anyhow::{Result};
+use diem_config::config::NodeConfig;
+use diem_sdk::{
+    client::BlockingClient,
+    crypto::PrivateKey,
+    transaction_builder::{Currency, TransactionFactory},
+    types::{
+        LocalAccount,
+    },
+};
+use diem_types::{
+    account_config,
+    chain_id::ChainId, transaction::authenticator::AuthenticationKey,
+};
+use generate_key::load_key;
+use shuffle_transaction_builder::framework::{
+    encode_create_parent_vasp_account_script_function,
+};
+use std::{
+    path::{Path, PathBuf},
+};
+
+
+
+pub fn handle(project_dir: PathBuf) -> Result<()> {
+
+    let project_ptr = project_dir.as_path();
+    let json_rpc_url = generate_json_rpc_url(project_ptr);
+
+    println!("Connecting to {}...", json_rpc_url);
+
+    let client = BlockingClient::new(json_rpc_url);
+    let mut root_account = generate_root_account(project_ptr, &client);
+    let new_account = generate_new_account();
+
+    println!("======new account {}", new_account.address());
+
+    // Create a new account.
+    print!("Create a new ParentVASP account (we cannot create a regular account right now)...");
+    let create_new_account_txn = root_account.sign_with_transaction_builder(
+        TransactionFactory::new(ChainId::test()).payload(
+            encode_create_parent_vasp_account_script_function(
+                Currency::XUS.type_tag(),
+                0,
+                new_account.address(),
+                new_account.authentication_key().prefix().to_vec(),
+                vec![],
+                false,
+            ),
+        ),
+    );
+    send(&client, create_new_account_txn)?;
+    println!("Success!");
+    Ok(())
+}
+
+
+pub fn generate_json_rpc_url(project_dir: &Path) -> String {
+    let config_path = project_dir.join("nodeconfig/0").join("node.yaml");
+    let config = NodeConfig::load(&config_path).expect("Failed to load \
+    NodeConfig from given project directory");
+    let json_rpc_url = format!("http://0.0.0.0:{}", config.json_rpc.address.port());
+    return json_rpc_url
+}
+
+pub fn generate_root_account(project_dir: &Path, client: &BlockingClient) -> LocalAccount {
```

Comment:
This name might be a bit confusing since we aren't generating the root account on-chain: it is published at genesis time already. What `LocalAccount::new` does on line 80 is creating a Rust struct that locally represents an account, whether the account exists on-chain or not. I'm okay with the name. Just wanna make sure we understand what the function does and doesn't do.